lib/onebox.rb

```diff
@@ -21,7 +21,8 @@ module Onebox
     allowed_ports: [80, 443],
     allowed_schemes: ["http", "https"],
     sanitize_config: Sanitize::Config::ONEBOX,
-    redirect_limit: 5
+    redirect_limit: 5,
+    disable_media_download_controls: false
```

Comment:
I'm not sure if this should be in the defaults. There are many types of oneboxes and not all are videos or audio oneboxes. For oneboxes that are not, this option does not make any sense. Is there a way where we can only apply this option for video and audio oneboxes only?

Reply:
I could just remove it from the default and then the video and audio onebox check can remain the same? There is no way to configure onebox apart from options passed into `Onebox.preview`. In core I am just changing the oneboxer to pass in this option:

```
disable_media_download_controls: SiteSetting.disable_onebox_media_download_controls
````

Reply:
Yes I think that'll be better to remove this from the default options and only pass the option in core when it is necessary.